lib/pushmi_pullyu/cli.rb

```diff
@@ -262,4 +263,22 @@ def start_server_as_daemon
     end
   end
 
+  def setup_db_connection
+    ActiveRecord::Base.establish_connection(database_configuration)
+  end
+
+  def database_configuration
```

Comment:
Not a huge problem, but what's your thoughts about simplifying this?

Can't see any reason why we need such granularity. Also I believe activerecord already does all this parsing and precedence on it's end already: https://edgeguides.rubyonrails.org/configuring.html#connection-preference 

This could just simply be passing in a url (we dont need pool/encoding/database_name/etc) or if we do want this granularity, I believe you can just pass in the database options hash (PushmiPullyu.options[:database]):
```ruby
def database_configuration
 PushmiPullyu.options[:database][:url]
end
```
This would also make configuration easier for henry/system admins as they can just copy the same `url:` from jupiter to pmpy. Instead of currently they being a little bit different for both applications

Reply:
Yes, I agreed. I moved the code from previous method directly but we should take the chance to simplify it. 

Will make the change.